GetJobber/atlantis
# feat(components): Allow GridCell' s to only be usable in Grids

## Motivations



## Changes



### Added

- 

### Changed

- Made it so `Grid.Cell` can only be used inside of a grid component

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Install pre-release.
- Add `Grid.Cell` that is not wrapped in a `Grid`. Verify an error occurs
- Add `Grid.Cell` that is wrapped in a `Grid`. Verify an error doesn't occur.

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
